src/Core/System/Resources/config/listing.xml

```diff
@@ -38,6 +38,15 @@
             <helpText lang="de-DE">Aktivieren um Bewertungen anzuzeigen</helpText>
         </input-field>
 
+        <input-field type="bool">
+            <name>enableStockManagement</name>
+            <label>Enable stock management</label>
+            <label lang="de-DE">Bestandsverwaltung ermöglichen</label>
+            <helpText>Activate to enable stock management. When disabled, product stock is managed by your administrator.</helpText>
```

Comment:
Text seems a bit strange. I would say it's more important to note what shopware is not doing: Something more like the description here: https://developer.shopware.com/docs/guides/hosting/configurations/shopware/stock.html#disable-stock-management-system.

Could also be catastrophic for a business when some random user disables stock management. What is the reason we want to make it configurable in the admin?

Reply:
Some shops, especially cloud instances, want to handle product stocks through their ERP system, and thus, we need to allow a way to disable this option (it was not possible in the SaaS context).
Would it make more sense to hide this setting in admin? so it can only be toggled using admin API? Or we can even introduce a dedicated API to toggle it with permission check
cc @mnaczenski

Reply:
I imagine it would be a one change right? I can't imagine a business would change this setting ever, maybe it should be an onboarding step?